packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.js

```diff
@@ -179,12 +182,10 @@ export default class QuanticResultQuickview extends LightningElement {
   }
 
   addRecentResult() {
-    getHeadlessEnginePromise(this.engineId).then((engine) => {
-      const {pushRecentResult} = this.headless.loadRecentResultsActions(engine);
-      engine.dispatch(
-        pushRecentResult(JSON.parse(JSON.stringify(this.result)))
-      );
-    });
+    const {pushRecentResult} = this.headless.loadRecentResultsActions(
+      this.engine
+    );
+    this.engine.dispatch(pushRecentResult(Object.create(this.result)));
```

Comment:
Interesting, I don't know enough about the potential caveats of doing this, but if it works and is extensively tested, should be ok